### ISSUE TITLE
(Very) slow build for LaTeX

### ISSUE THREAD

**Issue by arnaudmiribel** [tompollard/phd_thesis_markdown#62]
Hi there,

Just wanted to know whether it was normal or expected that the build time for the LaTeX (`make pdf`) is taking ~5 minutes to be done.
In your experience, are there good habits to overcome this ? I have quite a few attachments that may be heavy (36Mo of images) maybe that's the issue ?

Thanks

**Comment by Shannon-E-Taylor** [comment]
I have the same problem. I assume it's because of image size, as it started after I changed a bunch of figures from 96dpi to 600dpi. (I'm now using 300dpi :stuck_out_tongue_closed_eyes:  ) 

You could try using low resolution figures for drafts? ie 96dpi, and exporting them higher res once it's more polished. 

I've put a separate `make short` command in the Makefile to only output one chapter at a time. 

Just realised this is a year and a half old. Lol.  Hopefully this response is helpful for others :P

**Comment by tompollard** [comment]
> Just realised this is a year and a half old. Lol.

Yep, sorry, not doing a very good job of keeping on top of this repo! If anyone would like to help to maintain it, let me know.
